src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updateSubmission/UpdateSubmission.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updateSubmission.changes.Change;
+
+import java.util.List;
+
+public class UpdateSubmission {
+    @JsonProperty("type")
+    private String type = "OE02";
+
```

Comment:
Please check and confirm if this is indeed "type" or most likely to be "kind"

Reply:
This is in addition to the kind field in the FilingApi. I put this into the ODP just to have a "type" on the data. Can be as is.